slides/lesson1/solution/main_hard_code.tf

```diff
@@ -14,11 +16,20 @@ resource "aws_default_vpc" "default" {
   }
 }
 
+# Creation of the security group (SG) that
+# is going to be used for our instance
 resource "aws_security_group" "web" {
+  # Name to be assigned to the SG
   name        = "${data.aws_caller_identity.current.user_id}-web"
+  # Description of the SG
```

Comment:
About adding comments to the resources, what are your thoughts on adding those only at the top of the resource, and possibly also in properties which may not be self-explanatory?
My thoughts are that the `name` and `description` properties may not need a comment